src/RaisedButton/RaisedButton.js

```diff
@@ -251,21 +251,22 @@ class RaisedButton extends Component {
 
   state = {
     hovered: false,
+    keyboardFocused: false,
     touched: false,
     initialZDepth: 0,
     zDepth: 0,
   };
 
   componentWillMount() {
-    const zDepth = this.props.disabled ? 0 : 1;
+    const zDepth = (this.props.disabled) ? 0 : 1;
```

Comment:
Purely for readability - it makes the ternary more similar to an if, where the conditions are in parens. so when scanning the line it's clear that zDepth = (this.props.disabled) is not an assignment.